inc/namespace.php

```diff
@@ -29,6 +29,12 @@ function bootstrap() {
 		Block_Editor\bootstrap();
 	}
 
+	if ( $config['xmlrpc'] === false ) {
+		add_filter( 'xmlrpc_enabled', __NAMESPACE__ . '\\filter_xmlrpc_handler', 999 );
```

Comment:
You could use `__return_false` as the callback here I think, or did you find that those functions aren't loaded at this stage in an XMLRPC request?

Reply:
I was going to do that - but I saw this code https://github.com/humanmade/altis-cms/blob/master/inc/namespace.php#L139 , so thought it was a convention or for testing purposes?

Reply:
@pdewouters that makes sense - so in the case you linked to it's because WP hasn't fully loaded by the time that filter is run. In the case of the XMLRPC those functions are be available.